src/components/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { ReactElement } from "react";
+import '.././styles.css';
+
+const Footer = (): ReactElement => {
```

Comment:
I would add a comment here explaining what the component is for (even though it's obvious what it is rn, can be helpful in the future when component names get more complex!)